### DISCUSSION TITLE
http to https

### DISCUSSION THREAD

**Discussion by HAtherlolz** [aminalaee/sqladmin#627] [Q&A] [answered]
Hey guys, i have FastAPI app with sqlAlchemy and i'm trying to use sqladmin, it works fine in local host with http, but on a server that using ssl for https i got a issue. The issue is in requests from sqladmin does from http and browser blocks them. How can i make sqladmin works on https?

**Comment by HAtherlolz** [comment, marked as answer]
https://aminalaee.dev/sqladmin/cookbook/deployment_with_https/

**Reply by aminalaee** [reply]
Did that work?

**Reply by HAtherlolz** [reply]
yes, sorry for the stupid question, first I should have looked for existing ones.
